vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -249,13 +250,22 @@ func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 
 	dev, errAttach := d.ops.Attach(r.Name, nil)
 	if errAttach != nil {
-		log.WithFields(log.Fields{"name": r.Name,
-			"error": errAttach}).Error("Attach volume failed, removing the volume ")
-		errRemove := d.ops.Remove(r.Name, nil)
-		if errRemove != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": errRemove}).Warning("Remove volume failed ")
+		if strings.Contains(errAttach.Error(), internalError) {
```

Comment:
Yes, this may be a temporary approach to get the CI tests to pass. But looks like we have a more significant issue on the ESX service side that may be showing up this behavior in the client.